app/data_models/progress_store.py

```diff
@@ -125,12 +124,6 @@ def update_section_status(
                 self._progress[section_key].status = status
                 self._is_dirty = True
 
-        elif status == CompletionStatus.INDIVIDUAL_RESPONSE_REQUESTED:
```

Comment:
This could be improved now, since there is no `elif` anymore, you can return early etc., for example:
```python
    def update_section_status(
        self, status: CompletionStatus, section_key: SectionKey
    ) -> bool:
        """
        Updates the status of the Section or Repeating Blocks for a list item specified by the key based on the given section id and list item id.
        """
        if section_key not in self._progress:
            return False
        if self._progress[section_key].status != status:
            self._progress[section_key].status = status
            self._is_dirty = True
            return True
```

Reply:
Updated 👍